common/src/main/java/whocraft/tardis_refined/common/tardis/themes/console/InitiativeConsoleTheme.java

```diff
@@ -10,17 +10,36 @@ public class InitiativeConsoleTheme extends ConsoleThemeDetails {
-                new ControlSpecification(ConsoleControl.X, new Vector3f(-0.59999996f, 0.700003f, -0.5999999f), EntityDimensions.scalable(0.1f, 0.1f)),
-                new ControlSpecification(ConsoleControl.Y, new Vector3f(-0.75f, 0.7f, -0.44999972f), EntityDimensions.scalable(0.1f, 0.1f)),
-                new ControlSpecification(ConsoleControl.Z, new Vector3f(-0.84999996f, 0.700003f, -0.2999998f), EntityDimensions.scalable(0.1f, 0.1f)),
-                new ControlSpecification(ConsoleControl.INCREMENT, new Vector3f(-0.9499999f, 0.6f, 0.5000001f), EntityDimensions.scalable(0.1f, 0.1f)),
-                new ControlSpecification(ConsoleControl.ROTATE, new Vector3f(-0.3249999824911356F, 0.5750015135854483F, -1.049999713897705F), EntityDimensions.scalable(0.15F, 0.1F)),
-                new ControlSpecification(ConsoleControl.RANDOM, new Vector3f(-0.8499999f, 0.75f, 0.4500001f), EntityDimensions.scalable(0.1f, 0.1f)),
-                new ControlSpecification(ConsoleControl.MONITOR, new Vector3f(-0.5000000111758709F, 1.4000014774501324F, 0.32499980367720127F), EntityDimensions.scalable(0.3F, 0.25F)),
-                new ControlSpecification(ConsoleControl.DIMENSION, new Vector3f(-0.47499999962747097F, 0.9499969482421875F, -0.29999980330467224F), EntityDimensions.scalable(0.3F, 0.15F)),
-                new ControlSpecification(ConsoleControl.FAST_RETURN, new Vector3f(0.0F, 0.8500015139579773F, -0.5999995470046997F), EntityDimensions.scalable(0.2F, 0.15F))
+                new ControlSpecification(ConsoleControl.THROTTLE, new Vector3f(-0.012499999999999997f , 0.75f, 1.05f), EntityDimensions.scalable(0.125f , 0.125f)),
+                new ControlSpecification(ConsoleControl.X, new Vector3f(-0.66875f , 0.7187499999999998f, -0.621875f), EntityDimensions.scalable(0.125f , 0.125f)),
+                new ControlSpecification(ConsoleControl.Y, new Vector3f(-0.7500000000000003f , 0.7187499999999998f, -0.45f), EntityDimensions.scalable(0.12499999999999956f , 0.125f)),
+                new ControlSpecification(ConsoleControl.Z, new Vector3f(-0.8375000000000004f , 0.71875f, -0.29375f), EntityDimensions.scalable(0.12499999999999989f , 0.125f)),
+                new ControlSpecification(ConsoleControl.INCREMENT, new Vector3f(-0.3249999999999997f , 0.5624999999999987f, -1.043749999999999f), EntityDimensions.scalable(0.12500000000000003f , 0.12499999999999933f)),
+                new ControlSpecification(ConsoleControl.ROTATE, new Vector3f(-0.9999999999999993f , 0.625f, 0.49374999999999997f), EntityDimensions.scalable(0.12499999999999911f , 0.125f)),
+                new ControlSpecification(ConsoleControl.RANDOM, new Vector3f(0.3f , 0.5625f, -1.0562500000000001f), EntityDimensions.scalable(0.1249999999999997f , 0.12499999999999989f)),
+                new ControlSpecification(ConsoleControl.DOOR_TOGGLE, new Vector3f(0.037500000000000006f , 0.875f, -0.6375f), EntityDimensions.scalable(0.12499999999999999f , 0.125f)),
```

Comment:
This whole class is great, but can we possibly round down to 2 decimal places?

Reply:
Spoken about these. This will be done in the flight branch.